src/main/java/com/azure/cosmos/kafka/connect/sink/CosmosDBSinkTask.java

```diff
@@ -76,8 +73,7 @@ public void put(Collection<SinkRecord> records) {
 
                 Object recordValue;
                 if (record.value() instanceof Struct) {
-                    Map<String, Object> jsonMap = StructToJsonMap.toJsonMap((Struct) record.value());
-                    recordValue = mapper.convertValue(jsonMap, JsonNode.class);
+                    recordValue = StructToJsonMap.toJsonMap((Struct) record.value());

```

Comment:
@ryancrawcour I don't know what the reason for the conversion to `JsonNode` from `Map` is. Just leaving this as a `Map` for the cosmos client to upsert seems to work and it makes updating the map to insert the `id` field easier. Do you think this would break anything? Tests seem to be passing with this change.

Reply:
@ncliang not sure to be honest. i think it had something to do with what was being inserted in to cosmos db. i think we ended up with a string instead of json object, or something. it's too long ago to remember why i wrote it the way i did.  
let's leave it and see what happens. 

if it intros a regression we can always fix it (and add a better test to catch it next time)